api/tests/integration/edge_api/identities/test_edge_identity_featurestates_viewset.py

```diff
@@ -203,6 +319,7 @@ def test_edge_identities_create_featurestate(
     dynamo_wrapper_mock,
     feature,
     feature_name,
+    webhook_mock,
```

Comment:
How come we needed to add this fixture to these tests?

Reply:
Ah, because they call save on serializer : https://github.com/Flagsmith/flagsmith/blob/main/api/edge_api/identities/serializers.py#L174

Reply:
But we didn't add that in this PR so I'm not sure why it's necessary here? Ah, but is it throwing errors in the spawned threads?

Reply:
Yep, it's throwing error in the spawned therad